src/fidgets/farcaster/components/CastRow.tsx

```
@@ -725,7 +742,7 @@ const CastRowComponent = ({
             renderRecastBadge={renderRecastBadge}
             isDetailView={isFocused}
             userFid={userFid}
-            onSelectCast={onSelect}
+            onSelectCast={(hash) => onSelect && onSelect(hash, cast.author.username)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)